OrbitGl/GlCanvas.h

```diff
@@ -116,6 +117,23 @@ class GlCanvas {
 
   [[nodiscard]] PickingManager& GetPickingManager() { return picking_manager_; }
 
+  // Accessibility - providing default implementations, overridden in CaptureWindow
+  [[nodiscard]] int AccessibleChildCount() const override;
+  [[nodiscard]] const orbit_gl::GlAccessibleInterface* AccessibleChild(int index) const override;
+  [[nodiscard]] const GlAccessibleInterface* AccessibleParent() const override { return nullptr; }
+
+  // These will be handled by the AccessibilityAdapter, no need to implement them.
+  [[nodiscard]] std::string AccessibleName() const override { return ""; }
+  [[nodiscard]] orbit_gl::A11yRole AccessibleRole() const override {
+    return orbit_gl::A11yRole::Grouping;
+  }
+  [[nodiscard]] orbit_gl::A11yRect AccessibleLocalRect() const override {
+    return orbit_gl::A11yRect();
+  }
+  [[nodiscard]] orbit_gl::A11yState AccessibleState() const override {
+    return orbit_gl::A11yState();
+  }
```

Comment:
This is only required for GlCanvas, because it's a special case - but I also never really liked this and couldn't find a good design to avoid this. Reason is that those methods are handled by the OpenGlWidgetAccessible, but the Canvas can't know about this since it can't access it's "parent" (OpenGlWidget). It needs, however, to expose it's children (the TimeGraph)...

I could have the OpenGlWidgetAccessible handle the children directly and have it access the TimeGraph, but the OpenGlWidget returns a GlCanvas, the TimeGraph is defined in CaptureWindow...

But you're right that this isn't super nice. Let me see if I can come up with something better.